src/manifolds/MetricManifold.jl

```diff
@@ -37,44 +37,44 @@ giving the signature $(++...+-)$.
 """
 abstract type LorentzMetric <: Metric end
 
-@doc doc"""
+@doc raw"""
     RiemannianMetric <: Metric
 
 Abstract type for Riemannian metrics, a family of positive definite inner
-products. The positive definite property means that for $v \in T_x M$, the
-inner product $g(v, v) > 0$ whenever $v$ is not the zero vector.
+products. The positive definite property means that for $X  ∈ T_p ℳ$, the
+inner product $g(X, X) > 0$ whenever $X$ is not the zero vector.
 """
 abstract type RiemannianMetric <: Metric end
 
-function check_manifold_point(M::MetricManifold, x; kwargs...)
-    return check_manifold_point(M.manifold, x; kwargs...)
+function check_manifold_point(M::MetricManifold, p; kwargs...)
+    return check_manifold_point(M.manifold, p; kwargs...)
 end
 
-function check_tangent_vector(M::MetricManifold, x, v; kwargs...)
-    return check_tangent_vector(M.manifold, x, v; kwargs...)
+function check_tangent_vector(M::MetricManifold, p, X; kwargs...)
+    return check_tangent_vector(M.manifold, p, X; kwargs...)
 end
 
-@doc doc"""
-    christoffel_symbols_first(M::MetricManifold, x; backend=:default)
+@doc raw"""
+    christoffel_symbols_first(M::MetricManifold, p; backend=:default)
 
 Compute the Christoffel symbols of the first kind in local coordinates.
 The Christoffel symbols are (in Einstein summation convention)
 
-$\Gamma_{ijk} = \frac{1}{2} \left[g_{kj,i} + g_{ik,j} - g_{ij,k}\right],$
+$\Gamma_{ijk} = \frac{1}{2} \Bigl[g_{kj,i} + g_{ik,j} - g_{ij,k}\Bigr],$
 
-where $g_{ij,k}=\frac{\partial}{\partial x^k} g_{ij}$ is the coordinate
+where $g_{ij,k}=\frac{\partial}{\partial p^k} g_{ij}$ is the coordinate
```

Comment:
Can we use `∂`?

Reply:
Yes, good idea. I replaced all occurrences accordingly.